### ISSUE TITLE
DefPolicyPatch error on 10.0.17063.1000 x64

### ISSUE THREAD

**Issue by loyejaotdiqr47123** [llccd/RDPWrapOffsetFinder#5]
```
[10.0.17063.1000]
SingleUserPatch.x64=1
SingleUserOffset.x64=31739
SingleUserCode.x64=Zero
ERROR: DefPolicyPatch not found
LocalOnlyPatch.x64=1
LocalOnlyOffset.x64=92671
LocalOnlyCode.x64=jmpshort
SLInitHook.x64=1
SLInitOffset.x64=2318C
SLInitFunc.x64=New_CSLQuery_Initialize

[10.0.17063.1000-SLInit]
bServerSku.x64=F1378
bRemoteConnAllowed.x64=F2434
bFUSEnabled.x64=F2440
bAppServerAllowed.x64=F1380
bMultimonAllowed.x64=F2438
lMaxUserSessions.x64=F137C
ulMaxDebugSessions.x64=F243C
bInitialized.x64=F2430
```
![image](https://github.com/llccd/RDPWrapOffsetFinder/assets/102934154/2f18573f-34b7-489d-97f6-bb217f362012)

**Comment by loyejaotdiqr47123** [comment]
[termsrv_x64.zip](https://github.com/llccd/RDPWrapOffsetFinder/files/15141886/termsrv_x64.zip)

**Comment by loyejaotdiqr47123** [comment]
```
.text:00000001800111C2                 mov     ebx, 100h
.text:00000001800111C7                 mov     edi, 100h
.text:00000001800111CC                 nop
.text:00000001800111CD                 nop
.text:00000001800111CE                 cmp     ebx, edi
.text:00000001800111D0                 jz      loc_18003B100
```
Maybe so?
[https://github.com/sebaxakerhtc/rdpwrap.ini/commit/b76648d3cf7d1b9e7da32ea82b3ad884f0a06686](https://github.com/sebaxakerhtc/rdpwrap.ini/commit/b76648d3cf7d1b9e7da32ea82b3ad884f0a06686)

**Comment by loyejaotdiqr47123** [comment]
@llccd

**Comment by llccd** [comment]
Fixed in d0a704d. It uses `CDefPolicy_Query_edi_rcx`
